addons/web_editor/static/src/scss/web_editor.variables.scss

```diff
@@ -33,7 +33,7 @@ $o-we-border-color: $o-we-bg-light !default;
 
 $o-we-ace-color: #2F3129 !default;
 
-$o-we-handles-offset-to-hide: 10000px !default;
+$o-we-handles-offset-to-hide: 1.75e7px !default;
```

Comment:
> The value of 1.75e7px was chosen because it is the max value before the handle break on Firefox.
> The max threshold for chrome and edge is higher: 3e7px.

I was curious about this, so I tested with 1.76e7... and it seems to work? How do you arrive at the conclusion it breaks on Firefox with higher than 1.75e7. It's ok to try and use the maximum value available but then I think the value should be the exact one. But in the end, I would simply have augmented the value to 100000px. I can indeed imagine someone reaching 10000 (first case in 8 years)... but 100000px? ^^

Whatever you choose: add a small inline comment around the line to explain the value that was chosen, and in particular that 10000px was not enough :+1:

Reply:
It should be better now